functions.py

```diff
@@ -0,0 +1,41 @@
+import time
+import scipy
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy
+from scipy.optimize import curve_fit
+class interfaceInformation():
+  #Initialize with the clock system of the interface and robot
+  def __init__(self, clock): 
+    self.clock = clock
+    self.start_time = clock.time.time() #There will be a little bit of lag since this function will be called after interface is called
+   
+  def speed_estimates(self, C_d, density, frontal_area, angle_of_attack):
+    self.speed_left = math.sqrt((2*self.robot["left_motor"])/(C_d*density*frontal_area*sin(angle_of_attack)))
```

Comment:
Add spacing for readability and consider using a function to perform the calculations for readability as well.